closed/src/java.base/share/classes/jdk/crypto/jniprovider/NativeCrypto.java

```diff
@@ -64,12 +64,18 @@ public class NativeCrypto {
 
     private static final Cleaner ECKeyCleaner = CleanerFactory.cleaner();
 
+    private static final String javaHome =
+            GetPropertyAction.privilegedGetProperty("java.home");
```

Comment:
Moved.

Reply:
So, it will be something like the following, right?
```
String nativeLibName = System.getProperty("jdk.native.openssl.lib", "");
String javaHome = StaticProperty.javaHome();
```